test/e2e/network/networking_perf.go

```diff
@@ -70,10 +70,10 @@ func networkingIPerfTest(isIPv6 bool) {
 			func(n v1.Node) v1.PodSpec {
 				return v1.PodSpec{
 					Containers: []v1.Container{{
-						Name:  "iperf-server",
-						Image: imageutils.GetE2EImage(imageutils.Agnhost),
+						Name:    "iperf-server",
+						Image:   imageutils.GetE2EImage(imageutils.Agnhost),
+						Command: []string{"/bin/ash"},
```

Comment:
@aojea  This generally SGTM.
But why not use `/bin/sh` but `/bin/ash`?

```sh
# docker run -it --entrypoint ls  us.gcr.io/k8s-artifacts-prod/e2e-test-images/agnhost:2.13 /bin/
ash            dnsdomainname  ipcalc         mv             sh
base64         dumpkmap       kbd_mode       netstat        sleep
bash           echo           kill           nice           stat
bashbug        ed             linux32        pidof          stty
bbconfig       egrep          linux64        ping           su
busybox        false          ln             ping6          sync
cat            fatattr        login          pipe_progress  tar
catv           fdflush        ls             printenv       touch
chgrp          fgrep          lzop           ps             true
chmod          fsync          makemime       pwd            umount
chown          getopt         mkdir          reformime      uname
conspy         grep           mknod          rev            usleep
cp             gunzip         mktemp         rm             watch
date           gzip           more           rmdir          zcat
dd             hostname       mount          run-parts
df             ionice         mountpoint     sed
dmesg          iostat         mpstat         setserial
```
This image contains `sh` shell. Isn't it more standard?

Reply:
nice catch
I've used `ash` because didn't know that alpine has `sh` :sweat_smile: